rfcs/0000-User&HostIntegrationRFC.md

```diff
@@ -0,0 +1,248 @@
+asset.version	| keyword |	-           |	-	                | TBD	                    | Used primarily for 'Host' entities, the version or year associated with the asset.			
+asset.owner	    | keyword |	-           |	-	                | sourin.paul@elastic.co	| The primary user entity who owns the 'Host' asset.			
+asset.priority	| keyword |	Priority 1	| -                     | -                         | A priority classification for the asset obtained from outside this system, such as from some external CMDB or Directory service.			
+asset.criticality	| keyword |	Critical	| - | -                                         | A criticality classification obtained from outside this system, such as from some external CMDB or Directory service.			
+asset.business_unit	| keyword |	Analyst Experience	| - | -                                 | Business Unit associated with the asset (user or host).			
+asset.costCenter	| keyword |	Security - Protections | - | -                              | Cost Center associated with the asset (user or host).			
+asset.cost_center_hierarchy	| keyword |	Engineering	 | - | -                                | Additional cost center information associated with the asset (user or host).			
+asset.status	    | keyword         |	ACTIVE      | - | -                                 | Current status of the asset in the inventory datasource.			
+asset.last_status_change_date	| date |	June 5, 2023 @ 18:25:57.000	| - | -             | The most recent date/time when the asset.status was updated.			
+asset.create_date	            | date |	June 5, 2023 @ 18:25:57.001	| - | -             | For users, it's the hire date, for other assets, it's the in service date.			
+asset.end_date	                | date |	June 5, 2023 @ 18:25:57.002	| - | -             | For users, it's the termination date, for other assets, it's the out-of-service date.			
+asset.first_seen	            | date |	June 5, 2023 @ 18:25:57.003	| - | -             | The earliest date/time at which this asset was observed.  For assets added via config, this would remain empty until the asset was observed.			
+asset.last_seen	                | date |	June 5, 2023 @ 18:25:57.004	| - | -             | The most recent date/time at which this asset was observed.  Would remain empty until the asset was observed.			
+asset.last_updated	            | date |	June 5, 2023 @ 18:25:57.005	| - | -             | The most recent date/time this asset was updated in the inventory datasource.		
+asset.serial_number	            | keyword	| C02FG1G1MD6T	| - | -             |		Serial number of the asset.
+asset.tags	                    | keyword	  | watch, mdmaccess		| - | -             |	Tags assigned at the MDM.
+asset.assigned_users	          | keyword	  | user1@email.com, user2@email.com		| - | -             |	List of users assigned to the asset.
```

Comment:
I think the value of the `asset.owner` should always be included in this list. This will make is simpler to query for devices associated to a user because only one field needs to be searched. If you agree, then can you explicitly state in the description that the `asset.owner` value should always be included.

Reply:
I have incorporated your inputs in lines 52, 93, and 108.